gulpfile.js

```diff
@@ -82,9 +84,23 @@ gulp.task('build-css', css.bind(null, false));
 
 gulp.task('optimize-css', css.bind(null, true));
 
+gulp.task('jscs', function() {
+    process.stdout.write("JSCS the files...." + '\n');
```

Comment:
Should use "gutil.log" instead of "process.stdout.write" (I think - I have not seen process.stdout.write used in this context before)